source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required (VERSION 3.5)
+set(CMAKE_OSX_SYSROOT iphonesimulator11.2)
```

Comment:
Does this mean you can't build for a device?

Reply:
Yes, This is for symbol generations for vim

---
In reply to: [165780544](https://github.com/Microsoft/AdaptiveCards/pull/1097#discussion_r165780544) [](ancestors = 165780544)

Reply:
Wait, this isn't for building the project, but to be parsed by vim, the text editor? What does vim do with this file? Is that what all the cmake files in this project are for?

---
In reply to: [165792798](https://github.com/Microsoft/AdaptiveCards/pull/1097#discussion_r165792798) [](ancestors = 165792798,165780544)

Reply:
cmake will generate symbols, and a vim plugin can use it for intellisence. it works on mac. it will eventually work on WSL, then i don't have to work on mac anymore...

---
In reply to: [165795101](https://github.com/Microsoft/AdaptiveCards/pull/1097#discussion_r165795101) [](ancestors = 165795101,165792798,165780544)